modules/vba_documents/lib/vba_documents/upload_validator.rb

```diff
@@ -76,6 +76,7 @@ def validate_documents(parts, pdf_validator_options = VBADocuments::DocumentRequ
 
     def perfect_metadata(model, parts, timestamp)
       metadata = JSON.parse(parts['metadata'])
+      metadata['ICN'] = model.metadata['icn'] if model.metadata['icn'].present?
```

Comment:
Should we still send an `ICN` key with a `null` value if the ICN lookup wasn't successful, or would the EMMS API team prefer that we don't include the key if there is no ICN (sorry if you answered this elsewhere)?

Reply:
I did inquire in slack, if they say I should send in missing ICS as blanks, I'll address in another ticket if this PR gets merged